tests/testthat/test-gather_stock_time_series.R

```diff
@@ -0,0 +1,47 @@
+# test functions for gather_stock_time_series() function
+
+library(tidyverse)
+library(IntrinioSDK)
+library(testthat)
+
+# helper data
+api_key <- 'OjhlMjhjNTBmY2IyMWJiMWE0MTExYjQwNWZmZTVkZWM1'
+ticker <- 'AAPL'
+start_date <- '2011-01-01' 
+end_date <- '2019-12-10'
+
```

Comment:
After adding the exception handling parts in the function script, we can update corresponding tests here.

Reply:
same comments as above